meta-resin-common/recipes-containers/docker-disk/docker-disk.inc

```diff
@@ -35,7 +35,7 @@ do_compile () {
     losetup -f > /dev/null 2>&1 || bbfatal "docker-disk: Host must have at least one available loop device."
 
     docker build -t looper -f ${WORKDIR}/Dockerfile ${WORKDIR}
-    docker run --rm --privileged -e PARTITION_SIZE=${PARTITION_SIZE} -e TARGET_REPOSITORY=${TARGET_REPOSITORY} -e TARGET_TAG=${TARGET_TAG} -v /sys/fs/cgroup:/sys/fs/cgroup:ro -v ${B}:/export looper
+    docker run --rm --privileged -e PARTITION_SIZE=${PARTITION_SIZE} -e TARGET_REPOSITORY=${TARGET_REPOSITORY} -e TARGET_TAG=${TARGET_TAG} -v /sys/fs/cgroup:/sys/fs/cgroup:ro -v ${B}:/export -v ${DEPLOY_DIR_IMAGE}:/export2 looper
```

Comment:
Export Creates a data.img based on [block](https://github.com/ii/meta-resin/blob/e4fe50580b1a0bb70abe418d61e7db4905124e72/meta-resin-common/recipes-containers/docker-disk/files/entry.sh#L14) and makes an [ext4](https://github.com/ii/meta-resin/blob/e4fe50580b1a0bb70abe418d61e7db4905124e72/meta-resin-common/recipes-containers/docker-disk/files/entry.sh#L14) image to write the Resin AUFS [data](https://github.com/ii/meta-resin/blob/e4fe50580b1a0bb70abe418d61e7db4905124e72/meta-resin-common/recipes-containers/docker-disk/files/entry.sh#L18)
 which we can't mount with ubifs so we create a second Export ["Export2"](https://github.com/ii/meta-resin/blob/e4fe50580b1a0bb70abe418d61e7db4905124e72/meta-resin-common/recipes-containers/docker-disk/files/entry.sh#L49) based of just the Data and wait for the [mkubifs](https://gitlab.ii.org.nz/iichip/meta-chip/blob/ResinOS/classes/ubi_multi_vol-chip.bbclass#L154) stage to create a [volume](https://gitlab.ii.org.nz/iichip/meta-chip/blob/ResinOS/conf/machine/chip.conf#L31) out of it.